### DISCUSSION TITLE
Affiliate Agreement

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#41] [Added]
### Type name

Affiliate Agreement

### Alternative type names

Affiliate Policy

### Writer

service provider

### Audience

affiliate

### Object

~promotion, review, recommendations and commissions earning~
earning commissions by promoting the service https://github.com/OpenTermsArchive/terms-types/discussions/41#discussioncomment-10179185

### Examples

| Example | Service Name | URL | Title | Jurisdiction |
|:----|:----|:----|:------|:-------------|
| 1 | 4K Download | [URL 🔗](https://www.4kdownload.com/agreement/affiliate/2) | Affiliate Program Agreement | ❔ |
| 2 | FastestVPN | [URL 🔗](https://fastestvpn.com/affiliate-policy) | Affiliate Policy  | ❔ |
| 3 | GoZone WiFi | [URL 🔗](https://www.gozonewifi.com/terms/affiliate) | Affiliate Agreement | 🇺🇸 |
| 4 | Ivacy VPN | [URL 🔗](https://www.ivacy.com/legal/#afilitae-policy) | Affiliate Policy | 🇸🇬 |
| 5 | Traffic Authority | [URL 🔗](https://www.trafficauthority.net/legal/affiliate.php) | Affiliate Agreement | 🇺🇸 |

### References

- [Wikipedia page on Affiliate Marketing](https://en.wikipedia.org/wiki/Affiliate_marketing)

**Comment by MattiSG** [comment]
I would suggest:

- audience: third-party content authors
- object: earning commissions by promoting the service

**Reply by clementbiron** [reply]
> audience: third-party content authors

View in sync, the audience is not necessarily an author.

> object: earning commissions by promoting the service

I've updated it, thank you.

**Comment by MattiSG** [comment]
Suggested reference: [Wikipedia page on Affiliate Marketing](https://en.wikipedia.org/wiki/Affiliate_marketing).
